app/main.py

```diff
@@ -2,0 +1,33 @@
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared across all instances of `Animal` and its subclasses. If this is intended, it's fine. However, if each subclass should have its own `alive` list, consider moving the `alive` list to the `__init__` method or using a different approach.